docs/_includes/layouts/pages/element.11ty.ts

```diff
@@ -254,7 +254,7 @@ export default class ElementsPage extends Renderer<Context> {
     return content;
   }
 
-  async #renderInstallation({ doc, cdnVersion = 'v1-alpha' }: Context) {
+  async #renderInstallation({ doc, cdnVersion = 'v2' }: Context) {
```

Comment:
can we remove most of the installation-related content here and just link to the getting started page instead?

Reply:
Changed to "Import" heading with some text pointing to the getting started page for "Installation"